docker/prepare-environment.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+# Install git and patch commands, since we need them for Drush Make operations.
+apt-get update && \
+DEBIAN_FRONTEND=noninteractive apt-get -y install git patch
+
```

Comment:
Did you consider using a custom `Dockerfile`? Or wrapping it in some if statement testing whether the binaries have already been installed?